pkg/cmd/cli/cmd/deploy.go

```diff
@@ -365,7 +368,7 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig) error {
 				continue
 			}
 
-			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = deployapi.DeploymentCancelledAnnotationValue
+			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = unversioned.Now().Format(time.RFC3339)
```

Comment:
I am not sure about this change since we expect the previous value in other places I think.
https://github.com/openshift/origin/blob/5c6a1d473c6a5a1a15ede004f0fb86aeae6d339b/pkg/deploy/util/util.go#L419